README.md

```diff
@@ -116,11 +124,11 @@ Below is an incomplete list of feature that I believe are notable to this "base"
  - Custom [settings](src/core/globals.hpp) with deep compare if changes were made include auto saving
  - Clear and well structured source code
  - Includes a thread pool to queue tasks that shouldn't block the game thread, very similar to fiber pool
- - Updated natives.hpp from https://nativedb.spyral.dev
+ - Updated natives.hpp from https://nativedb.spyral.dev (We do not use fivem natives currently)
```

Comment:
Don't need to add this, YimMenu will never use FiveM natives. Only the ones officially in the game.